Themis/Models/Repository.swift

```diff
@@ -205,65 +200,65 @@ class Node: Hashable, ObservableObject {
+        "c": .c,
+        "h": .c,
+        "py": .python,
         "pyw": .python,
-        "java" : .java,
-        "swift" : .swift,
-        "rs" : .rust,
-        "gradle" : .gradle,
-        "jar" : .java,
-        "bat" : .bash,
-        "cmd" : .bash,
-        "sh" : .bash,
-        "js" : .javascript,
-        "dart" : .dart,
-        "groovy" : .groovy,
-        "hs" : .haskell,
-        "kt" : .kotlin,
-        "ktm" : .kotlin,
-        "scala" : .scala,
-        "sc" : .scala,
-        "ts" : .typescript,
-        "r" : .r,
-        "rb" : .ruby,
-        "sass" : .sass,
-        "scss" : .sass,
-        "spring" : .spring,
-        "sql" : .sqlite,
-        "ssh" : .ssh,
-        "svelte" : .svelte,
-        "twcss" : .tailwindcss,
-        "tf" : .tensorflow,
-        "tfvars" : .terraform,
-        "3js" : .threejs,
-        "typo3" : .typo3,
-        "code-workspace" : .vscode,
-        "vue" : .vuejs,
-        "webpack" : .webpack,
-        "xcodeproj" : .xcode,
-        "yarn" : .yarn,
-        "ino" : .arduino,
-        "cmake" : .cmake,
-        "cmake.in" : .cmake,
-        "coffee" : .coffeescript,
-        "css" : .css3,
-        "dockerfile" : .docker,
-        "graphql" : .graphql,
-        "grunt" : .grunt,
-        "gulp" : .gulp,
-        "html" : .html5,
-        "iml" : .intellij,
-        "jl" : .julia,
-        "ipynb" : .jupyter,
-        "tex" : .latex,
-        "lua" : .lua,
-        "md" : .markdown,
-        "m" : .matlab,
-        "mongo" : .mongodb,
-        "ml" : .ocaml,
-        "mli" : .ocaml
+        "java": .java,
+        "swift": .swift,
+        "rs": .rust,
+        "gradle": .gradle,
+        "jar": .java,
+        "bat": .bash,
+        "cmd": .bash,
+        "sh": .bash,
+        "js": .javascript,
+        "dart": .dart,
+        "groovy": .groovy,
+        "hs": .haskell,
+        "kt": .kotlin,
+        "ktm": .kotlin,
+        "scala": .scala,
+        "sc": .scala,
+        "ts": .typescript,
+        "r": .r,
+        "rb": .ruby,
+        "sass": .sass,
+        "scss": .sass,
+        "spring": .spring,
+        "sql": .sqlite,
+        "ssh": .ssh,
+        "svelte": .svelte,
+        "twcss": .tailwindcss,
+        "tf": .tensorflow,
+        "tfvars": .terraform,
+        "3js": .threejs,
+        "typo3": .typo3,
+        "code-workspace": .vscode,
+        "vue": .vuejs,
+        "webpack": .webpack,
+        "xcodeproj": .xcode,
+        "yarn": .yarn,
+        "ino": .arduino,
+        "cmake": .cmake,
+        "cmake.in": .cmake,
+        "coffee": .coffeescript,
+        "css": .css3,
+        "dockerfile": .docker,
+        "graphql": .graphql,
+        "grunt": .grunt,
+        "gulp": .gulp,
+        "html": .html5,
+        "iml": .intellij,
+        "jl": .julia,
+        "ipynb": .jupyter,
+        "tex": .latex,
+        "lua": .lua,
+        "md": .markdown,
+        "m": .matlab,
+        "mongo": .mongodb,
+        "ml": .ocaml,
+        "mli": .ocaml
```

Comment:
I Liked more the "md" : .markdown instead of "md": .markdown (space vs no space) but ok :((((

Reply:
fuck, I hate it 😂

Reply:
don't even know why this is happening